### PR TITLE
Bump CHANGELOG and VERSION for 2.1.4 release

### PR DESCRIPTION
**Pull request by pgwillia**
## Context

Prepare for 2.1.4 release

## What's New

Updated CHANGELOG and version.rb version.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,14 +8,31 @@ and releases in PushmiPullyu adheres to [Semantic Versioning](https://semver.org
 
 ## [Unreleased]
 
+## [2.1.4]
+
 ### Added
  - Bring in and enforce inclusive language cops [PR#469](https://github.com/ualbertalib/pushmi_pullyu/pull/469)
-
+
```
Comment (jefferya) [inline CHANGELOG.md:15]: There looks like an extra blank space character here?

File: Gemfile.lock [hunk 2]
```diff
@@ -246,4 +246,4 @@ DEPENDENCIES
   webmock (~> 3.24)
 
 BUNDLED WITH
-   2.3.12
+   2.4.9
```
Comment (jefferya) [inline Gemfile.lock:249]: Is there any advantage to using the same bundler version as jupiter? 

https://github.com/ualbertalib/jupiter/blob/caaa54e6932afd8c3c1a92b3a1a26eb6ee4d8b94/Gemfile.lock#L708
```ruby
   2.5.6
```

Comment (pgwillia) [reply Gemfile.lock:249]:
```
There are no versions of bundler (= 2.5.6) compatible with your Ruby & RubyGems
        bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.6.219.
```
I can see what might be required to bump the Ruby version, but I'll leave the bundler version and add a call out to the bundle version change in the Changelog.

Comment (jefferya) [reply Gemfile.lock:249]: Sounds good to me.

**Review by jefferya — APPROVED** [review]
👍

**Pull request merged**